test/test_return.py

```diff
@@ -178,6 +178,19 @@ def test_return_location_edit(self):
             == "/#%23foo%3Fbar"
         ), 'location with a variable with encoding'
 
+        assert 'success' in self.conf(
+            '""', 'routes/0/action/location'
+        ), 'location empty'
+        assert self.get()['headers']['Location'] == ''
+
+        assert 'success' in self.conf(
+            '"${host}"', 'routes/0/action/location'
+        ), 'location empty with variable'
```

Comment:
shouldn't that be `location with empty variable` as well?

Reply:
Both tests are testing the same thing, but I used slightly different names in case one of them fails it is easier to tell which one failed.

Reply:
All good :) Makes sense.